modules/tinymce/src/plugins/link/main/ts/core/DetectRtc.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import Editor from 'tinymce/core/api/Editor';
+
+export const hasRtcPlugin = (editor: Editor) => /(^|[ ,])rtc([, ]|$)/.test(editor.settings.plugins);
```

Comment:
Do we also want ` && PluginManager.get('rtc')` here, as that's the check we also include in the detect pro plugin modules?